examples/gltf_new/main.rs

```diff
@@ -230,7 +230,7 @@ fn main() -> Result<(), amethyst::Error> {
         .with_basic_renderer(
             path,
             DrawPbmSeparate::new()
-                .with_vertex_skinning()
+                //.with_vertex_skinning()
```

Comment:
Either remove it, or add a comment explaining why this line is commented and how it can be used if uncommented.